soc/aesc/vexriscv/nitrogen/Kconfig

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2025 Aesc Silicon
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_NITROGEN
```

Comment:
The path of this (and other) files should probably be `soc/aesc/nitrogen/Kconfig`